modules/maintainers/auxiliaries/deputize/request.pb.go

```diff
@@ -0,0 +1,576 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: proto/persistence/modules/maintainers/auxiliaries/deputize/v1beta1/request.proto
+
+package deputize
+
+import (
+	fmt "fmt"
+	_ "github.com/gogo/protobuf/gogoproto"
+	proto "github.com/gogo/protobuf/proto"
+	github_com_persistenceOne_persistenceSDK_schema_types "github.com/persistenceOne/persistenceSDK/schema/types"
+	io "io"
+	math "math"
+	math_bits "math/bits"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
+
+type auxiliaryRequest struct {
+	FromID               github_com_persistenceOne_persistenceSDK_schema_types.ID         `protobuf:"bytes,1,opt,name=fromID,proto3,customtype=github.com/persistenceOne/persistenceSDK/schema/types.ID" json:"fromID"valid:"required~required field fromID missing"`
+	ToID                 github_com_persistenceOne_persistenceSDK_schema_types.ID         `protobuf:"bytes,2,opt,name=toID,proto3,customtype=github.com/persistenceOne/persistenceSDK/schema/types.ID" json:"toID"valid:"required~required field toID missing"`
+	ClassificationID     github_com_persistenceOne_persistenceSDK_schema_types.ID         `protobuf:"bytes,3,opt,name=classificationID,proto3,customtype=github.com/persistenceOne/persistenceSDK/schema/types.ID" json:"classificationID"valid:"required~required field classificationID missing"`
+	MaintainedProperties github_com_persistenceOne_persistenceSDK_schema_types.Properties `protobuf:"bytes,4,opt,name=maintainedProperties,proto3,customtype=github.com/persistenceOne/persistenceSDK/schema/types.Properties" json:"maintainedProperties"valid:"required~required field maintainedProperties missing"`
+	AddMaintainer        bool                                                             `protobuf:"varint,5,opt,name=addMaintainer,proto3" json:"addMaintainer,omitempty"valid:"required~required field addMaintainer missing"`
+	RemoveMaintainer     bool                                                             `protobuf:"varint,6,opt,name=removeMaintainer,proto3" json:"removeMaintainer,omitempty"valid:"required~required field removeMaintainer missing"`
+	MutateMaintainer     bool                                                             `protobuf:"varint,7,opt,name=mutateMaintainer,proto3" json:"mutateMaintainer,omitempty"valid:"required~required field mutateMaintainer missing"`
+}
+
+func (m *auxiliaryRequest) Reset()         { *m = auxiliaryRequest{} }
+func (m *auxiliaryRequest) String() string { return proto.CompactTextString(m) }
```

Comment:
can remove pointer receiver whenever actual mutation of the receiver is not happening